README.md

```diff
@@ -1,5 +1,6 @@
-[GSConnect][ego] is a complete implementation of [KDE Connect][kdeconnect] for
-GNOME Shell with Nautilus, [Chrome][chrome] and [Firefox][firefox] integration.
+[GSConnect][ego] is a complete implementation **replacing** desktop [KDE
+Connect][kdeconnect]. It includes GNOME Shell, Nautilus, [Chrome][chrome] and
+[Firefox][firefox] integrations.
```

Comment:
I'd prefer if this were the same as the extensions.gnome.org text:

```
GSConnect is a complete implementation of KDE Connect especially for
GNOME Shell with Nautilus, Chrome and Firefox integration. It does
not rely on the KDE Connect desktop application and will not work
with it installed.
```

Reply:
OK I guess `will not work with it installed` is good enough.